### PR TITLE
Adding documentation to CWE-197 as part of #531

### PR DESCRIPTION
**Pull request by myteron**
Adding documentation to CWE-197 as part of #531

### CODE REVIEW

**Review by tommcd — APPROVED** [review]

File: docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+""" Compliant Code Example """
+counter = 0
+while counter <= 10:
+    value = counter/10
+    if value == 0.8:
+        print("we reached 0.8")
+        break
+    counter += 1
+```
+
```
[the remark this answers is not in the document]
Comment (myteron) [reply docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md:70]: good find. Will add this in.

File: docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+## Non-Compliant Code Example
+
+The `example02.py` code demonstrates more precision limites in floating numbers.
```
[the remark this answers is not in the document]
Comment (myteron) [reply docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md:73]: will change this back to what we had in confluence

File: docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+""" Compliant Code Example """
+counter = 1
+target = 10
+while counter <= target:
+    print(f"counter={counter / 10**16 :.20f}")
+    print(f" target={target / 10**16:.20f}")
+    counter += 1
+```
+
+## Defnitions
```
[the remark this answers is not in the document]
Comment (myteron) [reply docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md:123]: will fix

File: docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+```py
+""" Code Example """
+
+value = 0.0
+while value <= 1:
+    print(f"{type(value)} {value}")
+    value += 0.1
+```
+
+ **Output of exampl01.py:**
```
[the remark this answers is not in the document]
Comment (myteron) [reply docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/README.md:20]: will fix

**Review by s19110 — APPROVED** [review]

**Pull request merged**